app/src/main/java/nl/tudelft/b_b_w/view/AddBlockActivity.java

```diff
@@ -70,11 +70,15 @@ public void onAddBlock(View view) {
             String senderHash = senderHashText.getText().toString();
             String senderPublicKey = senderPublicKeyText.getText().toString();
 
+            //URGENCY: THIS NEED TO BE A SEPARATE INPUT FIELD FOR IBAN NUMBER
+            String senderIban = senderPublicKeyText.getText().toString();
+
+
 
             // create and add the block
             Block previous = handler.getLatestBlock(ownerName);
-
-            ConversionController conversionController = new ConversionController(ownerName, senderPublicKey, previous.getOwnHash(), senderHash);
+            int blockSeqNumber = previous.getSequenceNumber()+ 1;
+            ConversionController conversionController = new ConversionController(ownerName, blockSeqNumber, senderPublicKey, previous.getOwnHash(), senderHash, senderIban);
             String ownHash = conversionController.hashKey();
             Block block = new Block(
```

Comment:
The creation of blocks does not happen in DatabaseHandler, but BlockFactory. As @humanjas pointed out, the blocks are instantiated using the Block class, but this should happen in BlockFactory. So this issue isn't the same as stated in #15.